packages/x-date-pickers/src/models/pickers.ts

```diff
@@ -14,3 +14,22 @@ export interface PickerValidDateLookup {}
 export type PickerValidDate = keyof PickerValidDateLookup extends number
   ? any
   : PickerValidDateLookup[keyof PickerValidDateLookup];
+
+export interface PickerOwnerState<TValue> {
```

Comment:
I would be in favor of keeping the object as minimal as possible for now and add more properties as user request them.

But if you have another property that you think is important, I can of course add it :+1:

Reply:
It's a very good candidate for the `ownerState` indeed :ok_hand: 

I propose to treat it in a follow up, because I'm not sure we want to have it in its current shape.
We discussed following MD3 nomenclature (docked and modal) instead of mobile and desktop and this would impact the name of this property.